Southbank-Centre/southbankcentre.org
# 104731396 chore content switch env

To test:

- This branch has been deployed to the staging site where there is content available.
- Visit an event (e.g. http://staging.southbankcentre.org/event/baila-brazil-1627)
- Open the developer tools panel with the Network tab open
- Inspect the request for the event JSON
- See that it is requested from http://webcontent.staging.southbankcentre.org

BEFORE MERGING:

- The equivalent branch in the [angularjs-modules](https://github.com/Southbank-Centre/angularjs-modules) needs to be merged and the repo needs to be release to version 0.2.0.
- After that repo has been released, the version of angularjs-modules that gets installed into this app needs to be updated (it is currently using the feature branch version).
